docs/manipulation.rst

```diff
@@ -315,6 +315,31 @@ and True elsewhere. It is this way to be consistent with ``astropy.nddata``.
 .. note:: The mask attribute is the only attribute modified by ``snr_threshold()``. To
              retrieve the masked flux data use ``spectrum.masked.flux_masked``.
 
+Shifting
+--------
+
+To shift a spectrum in wavelength, one can use a construct like this:
```

Comment:
```suggestion

In addition to resampling, you may sometimes wish to simply shift
the ``spectral_axis`` of a spectrum (a la the ``specshift`` iraf task).
There is no explicit function for this because it is a basic transform of
just the ``spectral_axis``.  Therefore one can use a construct like this:
```

Reply:
Done